packages/react/src/Stack/Stack.module.css

```diff
@@ -180,42 +180,158 @@
  * Vertical alignment
  */
 
-.Stack-align-items--flex-end {
+.Stack--align-items-flex-end {
```

Comment:
I think we still need these, as they are required for the string value variant. Or you need to readjust the component import of this style to repoint at this new class.

I suspect it's causing this issue 👇 

![Screenshot 2023-01-23 at 11 18 36](https://user-images.githubusercontent.com/13340707/214027538-719f4234-1d90-442f-9b85-c03b261b492d.png)

Reply:
The problem came from the class names that were applied to the string value variants. Should be fixed now with  https://github.com/primer/brand/pull/164/commits/8f91aa2f756a675ffa687bd37cdd469842717f7a.